program-book-club.html

```diff
@@ -7,6 +7,11 @@
     <h1>Book Club Programme</h1>
 </div>
 <div class="container">
-    TODO
+    <div class="content-constrained">
+        <div class="section">
+            <p>What sets our book club apart are the lively conversations, valuable insights and an opportunity to connect with like-minded individuals. The books we read are selected by the community, offering a diverse selection of tech books and essential soft skills topics.</p>
+            <p>The book club meets monthly to share views, exchange ideas, and inspire personal and professional growth.</p> 
+            <p>Join our slack channel #book_club to be part of this vibrant community.</p>
```

Comment:
Please add the slack channel link here, as it is done for the other groups. 
```suggestion
            <p>Join our slack channel #prog_book_club to be part of this vibrant community.</p>
```

Reply:
added in last commit